### ISSUE TITLE
Add non-normative references as informational references 

### ISSUE THREAD

**Issue by manicprogrammer** [decentralized-identity/presentation-exchange#119]
Some terms and acronyms such as CHAPI, OIDC, DIDComm are not part of scope of the specification but are referenced in multiple places without definition or further reference. Propose to add an informative reference section in the appendix. 

Along with this proposal a PR will be provided that does this for the three acronyms above as a start point or to be denied and further edited or incorporated with issue #90 if desired.

**Comment by manicprogrammer** [comment]
There are code changes pending merge to master that impact the tooling used to render external references. At the request of @csuwildcat  I will wait for those changes and reapply my changes using the new tooling and issue a new PR at that time.

Closing the original PR #120 that got ahead of the new tooling. See the original commit in the fork for the proposed prior change.

**Comment by manicprogrammer** [comment]
@csuwildcat I am assuming the ref generator only works for w3 style references. At least one of the three (OIDC) I plan to add is not a w3 ref  (I don't think didcomm is either) how should those be handled? Manually in a subsection as I did it before?

**Comment by manicprogrammer** [comment]
I think the answer to that is probably you want to just leave them as inline links to the external site (which they already are) and not have them listed as external references. If that is the case then this entire issue just needs to go away as they are already compliant with that approach.

If that is the case you can close this with that comment.

**Comment by csuwildcat** [comment]
> @csuwildcat I am assuming the ref generator only works for w3 style references. At least one of the three (OIDC) I plan to add is not a w3 ref (I don't think didcomm is either) how should those be handled? Manually in a subsection as I did it before?

Because this tool allows you to create your own section with your own definitions as well as the augmented ones that are auto-pulled, you can mix the references together in the document and render them in the same place. Once the other PR is approved, I can add a few of the ones you're talking about so you can see the format. Can you add the link/spec example citations here and I will be sure to do that?

**Comment by manicprogrammer** [comment]
Will do. Forhcoming post.

**Comment by manicprogrammer** [comment]
I tried to follow the basic W3 premise but some fields don't apply for like a site where there is no author etc. I just did the best I could on each to apply all the W3 fields the most full being CHAPI since it is a W3 draft group

ref code
   name. authors/editor names, Editors. (if editors)
   site. publish sate. publish group.
   URL

`- **[CHAPI]** - 
     Credential Handler API 1.0. Dave Longley, Manu Sporny, Editors.
     W3C. 19 February 2020. Draft Community Group Report.
     URL: https://w3c-ccg.github.io/credential-handler-api/.
- **[DIDComm]** - 
     didcomm-messaging. @decentralized-identity, Owner. github.
     URL: https://github.com/decentralized-identity/didcomm-messaging.
- **[OIDC]** - 
     Welcome to OpenID Connect. OpenID.
     URL: https://openid.net/connect/.`

**Comment by csuwildcat** [comment]
Action: add informative refs, and specifically distinguish which are inform vs norm.
